Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -292,14 +310,24 @@ void SelectionChanged(object sender, SelectionChangedEventArgs e)
 			if (control == null || Element == null)
 				return;
 
-			var start = Element.CursorPosition;
+			int cursorPosition = Element.CursorPosition;
+
+			if (!_cursorPositionChangePending)
+			{
+				var start = cursorPosition;
 
-			if (control.SelectionStart != start)
-				ElementController?.SetValueFromRenderer(Entry.CursorPositionProperty, control.SelectionStart);
+				if (control.SelectionStart != start)
+					ElementController?.SetValueFromRenderer(Entry.CursorPositionProperty, control.SelectionStart);
```

Comment:
Should this get the same treatment as iOS with *_selectedTextRangeIsUpdating* so UpdateCursorSelection can just exit early like it does on iOS?

Reply:
Android does by way of setting `_selectionLengthChangePending` and `_cursorPositionChangePending` in `ElementPropertyChanged`. I didn't see any side effects in UWP, so it wasn't added there.

Reply:
I was thinking it'd be useful to short circuit like this during native changes to those fields
https://github.com/xamarin/Xamarin.Forms/pull/3402/files#diff-b8c74a29a641c60daba89e6908f411ffL338

Just so *UpdateCursorSelection* exits early when the user clicks around to make it less side effect prone down the road. None of that code needs to run with native changes to the cursor.  Maybe a general *_nativeCursorFeaturesUpdating* or something of that nature?